tests/modules/segmentation/test_create_graph.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+
+import ophys_etl.modules.segmentation.modules.create_graph.__main__ as cg
+
+
+@pytest.mark.parametrize(
+        "row_min, row_max, col_min, col_max, kernel, expected",
+        [
+            # 4 pixels, 6 edges
+            (0, 1, 0, 1, None, [[(0, 0), (0, 1)],
+                                [(0, 0), (1, 0)],
+                                [(1, 1), (1, 0)],
+                                [(1, 1), (0, 1)],
+                                [(1, 1), (0, 0)],
+                                [(0, 1), (1, 0)]]),
+            # 4 pixels, 6 edges, offset
+            (1, 2, 1, 2, None, [[(1, 1), (1, 2)],
+                                [(1, 1), (2, 1)],
+                                [(2, 2), (2, 1)],
+                                [(2, 2), (1, 2)],
+                                [(2, 2), (1, 1)],
+                                [(1, 2), (2, 1)]]),
+            # 4 pixels, 4 edges
+            (0, 1, 0, 1, [(-1, 0), (1, 0), (0, -1), (0, 1)],
+                [[(0, 0), (0, 1)],
+                 [(0, 0), (1, 0)],
+                 [(1, 1), (1, 0)],
+                 [(1, 1), (0, 1)]]),
+            # 6 pixels, 2 edges
```

Comment:
comment should say 4 pixels (which is what the test specifies; the comment just gave me a double-take)

Reply:
fixed